build.sbt

```diff
@@ -5,28 +5,29 @@ description := "Scala client to consume JSON home documents"
 
 organization := "de.kaufhof"
 
-version := "2.2.0"
+version := "2.2.1"
```

Comment:
Changed it. Though i'm not sure if this should be part of the PR at all or be part of a release commit.